### PR TITLE
fix AbstractFloat support

### PR DESCRIPTION
**Pull request by jw3126**

### CODE REVIEW

File: src/src.jl [hunk 1]
```diff
@@ -131,7 +131,7 @@ function push_fsize_zero(pivot::Array{Float64, 1}, container::MBContainer)
 fsize != 1, calculate the miniball center and squared radius using the current
 pivot element.
 """
- function push_fsize_not_zero(pivot::Array{Float64, 1}, container::MBContainer)
+function push_fsize_not_zero(pivot::Array{<:AbstractFloat, 1}, container::MBContainer)
     epsilon = pow_2(1e-21)
```
Comment (jw3126) [inline src/src.jl]: Not sure how this number generalizes to `AbstractFloat`?

Comment (jw3126) [reply src/src.jl]: Exactly the same fuzz tests pass if I set this number to `0`.

Comment (jw3126) [reply src/src.jl]: And the code takes exactly the same branches, too.

**Comment by TeroFrondelius** [comment]
@jw3126 can I merge this?

**Comment by jw3126** [comment]
Yes you can merge this!

**Pull request merged**